### DISCUSSION TITLE
[General] Broken Link in Repository Description

### DISCUSSION THREAD

**Discussion by aolangch** [zkSync-Community-Hub/zksync-developers#573] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

I found a broken link in the repository description. The link to the tutorials (era.zksync.io/docs/dev/tutorials/) is not working. Please update the link or provide an alternative.

### Repo Link (Optional)

https://github.com/zkSync-Community-Hub/tutorials

### Additional Details

![screencapture-docs-zksync-io-docs-dev-tutorials-2024-06-11-06_37_42](https://github.com/zkSync-Community-Hub/zksync-developers/assets/49438860/09571a17-6b00-429a-9df5-8be3e8b9e882)

**Comment by bxpana** [comment]
Thanks for the heads up. We'll make sure to update that.

**Reply by bxpana** [reply]
Fixed, thanks again. https://github.com/zkSync-Community-Hub/tutorials/blob/main/README.md
